examples/cloud_storage_stream_inference/Download_Transformer_models.py

```diff
@@ -0,0 +1,77 @@
+import transformers
```

Comment:
Would it be possible to use the original file? Not a huge fan of this kind of code duplication since the code will quickly rot 

I'd prefer if we used one of the default handlers for this example, even something like denset from our getting started guide

Reply:
Removed code duplication